Index: app/src/main/java/com/mancode/financetracker/AccountFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/mancode/financetracker/AccountFragment.java	(date 1501443351000)
+++ app/src/main/java/com/mancode/financetracker/AccountFragment.java	(revision )
@@ -69,7 +69,9 @@
             } else {
                 rView.setLayoutManager(new GridLayoutManager(context, mColumnCount));
             }
-            rView.setAdapter(new AccountRecyclerViewAdapter(DatabaseHelper.AccountItem.ITEMS, mListener));
+            rView.setAdapter(new AccountRecyclerViewAdapter(
+                    DatabaseHelper.getInstance(getActivity().getApplicationContext()).getAllAccountsFromDB(),
+                    mListener));
         }
 
         FloatingActionButton fab = (FloatingActionButton) view.findViewById(R.id.fab);
@@ -119,10 +121,14 @@
         FragmentManager fragmentManager = getFragmentManager();
         AddAccountFragment addAccountFragment = new AddAccountFragment();
 
+//        FragmentTransaction transaction = fragmentManager.beginTransaction();
+//        transaction.setTransition(FragmentTransaction.TRANSIT_FRAGMENT_OPEN);
+//        transaction.addToBackStack(null);
+//        addAccountFragment.show(transaction, "addAccountFragment");
+//
         FragmentTransaction transaction = fragmentManager.beginTransaction();
-        transaction.setTransition(FragmentTransaction.TRANSIT_FRAGMENT_OPEN);
-        transaction.add(android.R.id.content, addAccountFragment);
-        transaction.addToBackStack(null);
-        transaction.commit();
+//        transaction.setTransition(FragmentTransaction.TRANSIT_FRAGMENT_OPEN);
+        transaction.setCustomAnimations(android.R.animator.fade_in, android.R.animator.fade_out, android.R.animator.fade_in, android.R.animator.fade_out);
+        transaction.add(android.R.id.content, addAccountFragment).addToBackStack(null).commit();
     }
 }
